bitbots_blackboard/src/bitbots_blackboard/capsules/team_data_capsule.py

```diff
@@ -6,13 +6,16 @@
 
 import rospy
 from humanoid_league_msgs.msg import Strategy, TeamData
+from collections import defaultdict
 
 
 class TeamDataCapsule:
     def __init__(self):
         self.bot_id = rospy.get_param("bot_id", 1)
         self.strategy_sender = None  # type: rospy.Publisher
         self.team_data = TeamData()
+        self.team_strategy = defaultdict(lambda: Strategy.ROLE_IDLING)
+        self.times_to_ball = defaultdict(None)
```

Comment:
None for a defaultdict is just a dict. Since the value of all other robots should be maximal to never choose them for the rank, something like `defaultdict(lambda: np.inf)` would make sense.